modules/dxp/apps/scim/scim-rest-impl/src/main/java/com/liferay/scim/rest/internal/manager/UserManagerImpl.java

```diff
@@ -535,6 +717,85 @@ private ScimUser _getScimUser(long companyId, long userId)
 		return _toScimUser(portalUser);
 	}
 
+	private UserGroup _getUserGroup(long companyId, long groupId)
+		throws AbstractCharonException {
```

Comment:
`long groupId` should be `long userGroupId` because it relates to the portal UserGroup model?